frontend/helpers/path-helpers.ts

```diff
@@ -46,3 +47,9 @@ export function productListPath(
       }
    }
 }
+
+export function ifixitOriginFromHost(host?: string): string {
```

Comment:
Let's have this take in a `context` or a `request` have it do:

`const host = headers['x-ifixit-forwarded-host`] || headers['x-forwarded-host']`

Does that seem reasonable?

Reply:
Yea seems like an improvement, so the caller doesn't have to drill down in to the context

Reply:
https://github.com/iFixit/react-commerce/pull/1025/commits/6cb45de738a3463eb273e19b39e725017a6bd274